src/endpoint.js

```diff
@@ -74,6 +76,19 @@ module.exports = class AbstractEndpoint {
     return this
   }
 
+  //turns on auto-batching for this endpoint
+  enableAutoBatch (interval) {
+    if (this._autoBatch === null) {
+      this._autoBatch = {
+        interval: interval || 1000,
+        set: new Set(),
```

Comment:
Rename into something that explains better what it does, e.g. `ids`

Reply:
New name: idsForNextBatch